### PR TITLE
Checklist upgrade

### PR DESCRIPTION
**Pull request by wackerow**
Restructured checklist
- [x] Breaks into three main sections, Preparation, Setup, and Monitoring/Maintenance
- [x] Adds and updates content
- [x] Adds testnet suggestion
- [x] Provides link to checklist from initial disclaimers/advisories section
- [x] Add section links at top

@samajammin @ryancreatescopy Take a look at this and let me know what you think of the changes. Modeled after Figma notes.

### CODE REVIEW

File: src/pages/Checklist/index.tsx [hunk 1]
```diff
@@ -168,78 +177,186 @@ export const Checklist = () => {
...
+          <Heading level={4}>
+            <FormattedMessage defaultMessage="Recommendation disclaimer" />
+          </Heading>
           <ul className="sub-checklist-item">
             <li className="py5">
               <Text>
                 <FormattedMessage
-                  defaultMessage="In order to process incoming validator deposits from the eth1
-                    chain, you will need to run an eth1 client in parallel to your
+                  defaultMessage="Please note that hardware suggestions are an ever evolving target. Current
```
Comment (ryancreatescopy) [inline src/pages/Checklist/index.tsx:213]: Let's remove the "pleases" in this page. I think it gets overused in UI copy and isn't really needed.

File: src/pages/Checklist/index.tsx [hunk 2]
```diff
@@ -498,80 +619,90 @@ export const Checklist = () => {
...
-                    connected it to my Beacon Node via HTTP/gRPC API(s)."
-                />
+                <FormattedMessage defaultMessage="I have started running my validator client" />
               </Text>
             }
           />
         </section>
+        <SectionHeader>
+          <Heading level={3}>
+            <FormattedMessage defaultMessage="Section 3 - Monitoring and Maintenance" />
```
Comment (ryancreatescopy) [inline src/pages/Checklist/index.tsx:698]: I think Section 3, could do with a subtitle – why is this important?

File: src/pages/Checklist/index.tsx [hunk 3]
```diff
@@ -168,78 +177,186 @@ export const Checklist = () => {
...
+            <li className="py5">
+              <Text>
+                <FormattedMessage defaultMessage="Note: syncing your Eth1 client may take a few days in the worst-case scenario" />
+              </Text>
+            </li>
+          </ul>
+        </section>
+        <SectionHeader>
+          <Heading level={3}>
+            <FormattedMessage defaultMessage="Section 2 - Setup" />
```
Comment (ryancreatescopy) [inline src/pages/Checklist/index.tsx:303]: Love the new sections but I think we should frame the copy around the stage of the process. "Before you start", "During setup", "After deposit" would be a clearer way to frame when to follow these steps IMO.

**Review by ryancreatescopy — CHANGES_REQUESTED** [review]
Love this update Paul. Just some minor comments. I defer to @CarlBeek to review the actual content, but overall the new structure is really strong in my opinion.

File: src/pages/Checklist/index.tsx [hunk 4]
```diff
@@ -168,78 +177,186 @@ export const Checklist = () => {
...
+                  defaultMessage="Please note that hardware suggestions are an ever evolving target. Current
+                    minimum requirements are likely to increase by an order of magnitude after the merge and 
+                    introduction of shard chains. Please do your own research before depositing funds."
+                />
+              </Text>
+            </li>
+          </ul>
+        </section>
+        <section>
+          <Heading level={3}>
```
Comment (ryancreatescopy) [inline src/pages/Checklist/index.tsx:222]: Link to this from the new homepage "Become a validator" steps?

File: src/pages/Checklist/index.tsx [hunk 5]
```diff
@@ -168,78 +177,186 @@ export const Checklist = () => {
...
+              <Text>
+                <FormattedMessage defaultMessage="Note: syncing your Eth1 client may take a few days in the worst-case scenario" />
+              </Text>
+            </li>
+          </ul>
+        </section>
+        <SectionHeader>
+          <Heading level={3}>
+            <FormattedMessage defaultMessage="Section 2 - Setup" />
+          </Heading>
```
Comment (ryancreatescopy) [inline src/pages/Checklist/index.tsx:304]: Again, maybe a subtitle that explains this checklist is designed to help you while you go through the deposit process.

File: src/pages/Checklist/index.tsx (hunk 4 above)
Comment (wackerow) [reply src/pages/Checklist/index.tsx:222]: Thinking here was "Become a validator" takes you into the disclaimer overview, of which this checklist is now one of the items presented to new users. Meanwhile the "Thinking about staking?" button leads to the checklist.

Makes me think linking to the FAQ from the end of the checklist may not be a bad idea

File: src/pages/Checklist/index.tsx [hunk 6]
```diff
@@ -168,78 +177,186 @@ export const Checklist = () => {
...
+                <Link primary inline to="https://reddit.com/r/ethstaker">
+                  Reddit
+                </Link>
+              ),
+            }}
+            description="{variables} are social media platform links to Discord and Reddit (do not translate names)"
+          />
+        </Text>
       </Subtitle>
       <ChecklistPageStyles>
```
[the remark this answers is not in the document]
Comment (wackerow) [reply src/pages/Checklist/index.tsx:281]: @ryancreatescopy Gave this a shot, lemme know what you think

**Comment by wackerow** [comment]
@ryancreatescopy Implemented your suggestions and updated Figma. Take a peak and let me know your thoughts

File: src/pages/Acknowledgements/pageContent.tsx [hunk 7]
```diff
@@ -267,6 +267,22 @@ export const pageContent = {
...
     ),
   },
+  [AcknowledgementIdsEnum.checklist]: {
+    title: 'Staking Checklist',
+    content: (
+      <>
+        <Text size="medium" className="my10">
+          <FormattedMessage defaultMessage="Please review the staking checklist prior to proceeding. Use this as a guide to check off tasks as you complete validator setup." />
+        </Text>
+        <Link inline to="/checklist" className="my10" primary>
```
Comment (samajammin) [inline src/pages/Acknowledgements/pageContent.tsx:277]:
```suggestion
        <Link inline shouldOpenNewTab={true} to="/checklist" className="my10" primary>
```

File: src/pages/Checklist/index.tsx [hunk 8]
```diff
@@ -188,58 +318,176 @@ export const Checklist = () => {
             <li className="py5">
               <Text>
                 <FormattedMessage
-                  defaultMessage="For example, you need at least ~400 GB SSD to run Geth sync on
-                    {mainnet} in February 2021, assuming you move the {ancient} directory on a
-                    different drive. Have in mind that Geth {chaindata} grows with ~1 GB per day,
-                    so you should account for enough space on your SSD and HDD until you run
-                    maintenance on the node."
+                  defaultMessage="As of {date}, ~400GB is needed for the Eth1 mainnet chain data alone (growing at ~1GB/day)."
```
Comment (samajammin) [inline src/pages/Checklist/index.tsx:321]: FYI @wackerow @ryancreatescopy I refactored these use of dates to use the `<FormattedDate/>` component, which should automatically translate & format dates depending on the current language. If you notice any other use of dates, please follow this pattern.

**Review by samajammin — APPROVED** [review]
Great work @wackerow! LGTM. Added date formatting (see comment) & some small styling (warning banner for the hardware disclaimer).

**Pull request merged**